### PR TITLE
Fix link and head issue

### PR DESCRIPTION
**Pull request by kevinthesun**
Fix broken links and quick fix for heading issue.

### CODE REVIEW

**Comment by piiswrong** [comment]
Could you point the broken links to the updated location instead of to the homepage?

**Comment by kevinthesun** [comment]
Some links point to home page since the pages which they should point to are still missing.

File: docs/tutorials/computer_vision/imagenet_full.md [hunk 1]
```diff
@@ -40,9 +40,9 @@ After packing, together with threaded buffer iterator, we can simply achieve an
...
-This however, creates a challenge for GPU memory. As GTX980 only have 4G of GPU RAM. We really need to minimize the memory consumption to fit larger batch-size into the training. To solve this problem we use the techniques such as node memory reuse, and inplace optimization, which reduces the memory consumption by half, more details can be found in  [memory optimization note](http://mxnet.readthedocs.org/en/latest/developer-guide/note_memory.html)
+This however, creates a challenge for GPU memory. As GTX980 only have 4G of GPU RAM. We really need to minimize the memory consumption to fit larger batch-size into the training. To solve this problem we use the techniques such as node memory reuse, and inplace optimization, which reduces the memory consumption by half, more details can be found in  [memory optimization note](http://mxnet.io)
```
Comment (piiswrong) [inline docs/tutorials/computer_vision/imagenet_full.md]: http://mxnet.io/architecture/note_memory.html

File: docs/get_started/setup.md [hunk 2]
```diff
@@ -406,7 +406,7 @@ You might want to add this command to your ```~/.bashrc``` file. If you do, you
 Pkg.add("MXNet")
 ```
 
-For more details about installing and using MXNet with Julia, see the [MXNet Julia documentation](http://mxnetjl.readthedocs.org/en/latest/user-guide/install.html)
+For more details about installing and using MXNet with Julia, see the [MXNet Julia documentation](http://dmlc.ml/MXNet.jl/latest/)
```
Comment (piiswrong) [inline docs/get_started/setup.md]: http://dmlc.ml/MXNet.jl/latest/user-guide/install/

File: docs/tutorials/computer_vision/imagenet_full.md [hunk 3]
```diff
@@ -40,9 +40,9 @@ After packing, together with threaded buffer iterator, we can simply achieve an
...
 
-Finally, we cannot train the model using a single GPU because this is a really large net, and a lot of data. We use data parallelism on four GPUs to train this model, which involves smart synchronization of parameters between different GPUs, and overlap the communication and computation. A [runtime denpdency engine](https://mxnet.readthedocs.org/en/latest/developer-guide/note_engine.html) is used to simplify this task, allowing us to run the training at around 170 images/sec.
+Finally, we cannot train the model using a single GPU because this is a really large net, and a lot of data. We use data parallelism on four GPUs to train this model, which involves smart synchronization of parameters between different GPUs, and overlap the communication and computation. A [runtime denpdency engine](http://mxnet.io) is used to simplify this task, allowing us to run the training at around 170 images/sec.
```
Comment (piiswrong) [inline docs/tutorials/computer_vision/imagenet_full.md]: http://mxnet.io/architecture/note_engine.html

File: docs/tutorials/r/CallbackFunctionTutorial.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ which can very useful in model training.
 
 This tutorial is written in Rmarkdown.
 
-- You can directly view the hosted version of the tutorial from [MXNet R Document](http://mxnet.io/api/r/CallbackFunctionTutorial.html)
```
Comment (piiswrong) [inline docs/tutorials/r/CallbackFunctionTutorial.md]: http://mxnet.io/tutorials/r/CallbackFunctionTutorial.html

File: docs/tutorials/r/charRnnModel.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ Data can be found at [here](https://github.com/dmlc/web-data/tree/master/mxnet/t
 Preface
 -------
 This tutorial is written in Rmarkdown.
-- You can directly view the hosted version of the tutorial from [MXNet R Document](http://mxnet.io/api/r/CharRnnModel.html)
+- You can directly view the hosted version of the tutorial from [MXNet R Document](http://mxnet.io)
```
Comment (piiswrong) [inline docs/tutorials/r/charRnnModel.md]: http://mxnet.io/tutorials/r/charRnnModel.html

File: docs/tutorials/r/classifyRealImageWithPretrainedModel.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ This model gives the recent state-of-art prediction accuracy on image net datase
 Preface
 -------
 This tutorial is written in Rmarkdown.
-- You can directly view the hosted version of the tutorial from [MXNet R Document](http://mxnet.io/api/r/classifyRealImageWithPretrainedModel.html)
```
Comment (piiswrong) [inline docs/tutorials/r/classifyRealImageWithPretrainedModel.md]: http://mxnet.io/tutorials/r/classifyRealImageWithPretrainedModel.html

File: docs/tutorials/r/fiveMinutesNeuralNetwork.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ We will show you how to do classification and regression tasks respectively. The
 Preface
 -------
 This tutorial is written in Rmarkdown.
-- You can directly view the hosted version of the tutorial from [MXNet R Document](http://mxnet.io/api/r/fiveMinutesNeuralNetwork.html)
```
Comment (piiswrong) [inline docs/tutorials/r/fiveMinutesNeuralNetwork.md]: http://mxnet.io/tutorials/r/fiveMinutesNeuralNetwork.html

File: docs/tutorials/r/mnistCompetition.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ Handwritten Digits Classification Competition
 We will present the basic usage of [MXNet](https://github.com/dmlc/mxnet/tree/master/R-package) to compete in this challenge.
 
 This tutorial is written in Rmarkdown. You can download the source [here](https://github.com/dmlc/mxnet/blob/master/R-package/vignettes/mnistCompetition.Rmd) and view a
-hosted version of tutorial [here](http://mxnet.io/api/r/mnistCompetition.html).
+hosted version of tutorial [here](http://mxnet.io).
```
Comment (piiswrong) [inline docs/tutorials/r/mnistCompetition.md]: http://mxnet.io/tutorials/r/mnistCompetition.html

File: docs/zh/api/r/CallbackFunctionTutorial.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 本文将概述我们如何在模型训练的时候使用或者自定义一些回调函数。本教程使用 Rmarkdown 编写。
 
-- 你可以直接看我们在主站上的教程： [MXNet R Document](http://mxnet.io/api/r/CallbackFunctionTutorial.html)
```
Comment (piiswrong) [inline docs/zh/api/r/CallbackFunctionTutorial.md]: http://mxnet.io/tutorials/r/CallbackFunctionTutorial.html

File: docs/zh/api/r/fiveMinutesNeuralNetwork.md [hunk 10]
```diff
@@ -8,7 +8,7 @@
 
 本教程由Rmd编辑完成。
 
-- 你可以直接访问主站版本的教程：[MXNet R Document](http://mxnet.io/api/r/fiveMinutesNeuralNetwork.html)
+- 你可以直接访问主站版本的教程：[MXNet R Document](http://mxnet.io)
 - 你也可以从[这里](https://github.com/dmlc/mxnet/blob/master/R-package/vignettes/fiveMinutesNeuralNetwork.Rmd) 下载到Rmarkdown源文件
```
Comment (piiswrong) [inline docs/zh/api/r/fiveMinutesNeuralNetwork.md]: http://mxnet.io/tutorials/r/fiveMinutesNeuralNetwork.html

Comment (piiswrong) [inline docs/zh/api/r/fiveMinutesNeuralNetwork.md]: the files are moved. point them to the updated locations in my comment

**Pull request closed without merging**